src/osd/osd_types.h

```diff
@@ -2962,6 +2962,11 @@ struct pg_log_t {
   mempool::osd::list<pg_log_entry_t> log;  // the actual log.
   
   pg_log_t() = default;
+  pg_log_t(const pg_log_t& other) = default;
+  pg_log_t& operator=(const pg_log_t& other) = default;
+  pg_log_t(pg_log_t&& other) = default;
+  pg_log_t& operator=(pg_log_t&& other) = default;
+
```

Comment:
it's not clear to me what was preventing the compiler from generating these before?